CONTRIBUTING.md

```diff
@@ -33,7 +33,15 @@ Follow these steps to build a new version of the plugin:
 
 ### Environment setup
 
-Make sure you can build the plugin by following the steps above.
+First, make sure you can build the plugin by following the steps above.
+
+Then, make sure you have installed ```Gulp```, ```Mocha```, and ```Mocha JUnit Reporter``` through npm.
+
+```
+npm install -g gulp
+npm install -g mocha
+npm install -g mocha-junit-reporter
```

Comment:
so it works with node_modules/.bin/mocha but you would have to call node_modules/.bin/gulp if you want that to work off devdependencies alone, so I will keep the gulp install in there and then tell the user to do an npm install